### PR TITLE
Fix bug with bicubic interpolation

### LINKED ISSUE #11: resize using INTERPOLATION_BICUBIC has problem
I using the resize method at INTERPOLATION_BICUBIC mode, the result is diffrent with python api。

c++ code ：
  cv::Mat src = cv::imread("sample.jpg")
  int new_w = 736;
  int new_h = 391;
  cv::Size resize_size = cv::Size(new_w, new_h);
  PillowResize::InterpolationMethods interpolation =
      PillowResize::InterpolationMethods::INTERPOLATION_BICUBIC;
  cv::Mat cpp_resize= PillowResize::resize(src, resize_size, interpolation);

python code:
  src = cv2.imread("sample.jpg")
  new_w = 736
  new_h = 391
  src_ = Image.fromarray(src)
  resized_image= src_.resize((new_w, new_h), Image.BICUBIC)

image

### PR DESCRIPTION
**Pull request by marcov868**
Closes https://github.com/zurutech/pillow-resize/issues/11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,17 @@ cmake ..
 make
 ```
 
+### Test
+If you want to run also the tests, you need to install also [`gtest`](https://github.com/google/googletest).
```
Comment (iLeW) [inline README.md]:
```suggestion
If you want to run also the tests, you need to install [`gtest`](https://github.com/google/googletest).
```

File: tests/src/PillowResize/ResizeTest.cc [hunk 2]
```diff
@@ -0,0 +1,100 @@
+/*
+* Copyright 2022 Zuru Tech HK Limited
```
Comment (galeone) [inline tests/src/PillowResize/ResizeTest.cc]: we are in 2023 :rofl:

Comment (marcov868) [reply tests/src/PillowResize/ResizeTest.cc]: 🤦‍♂️

**Comment by galeone** [comment]
Now that there are tests, it's worth adding a CI file using Github actions. I haven't looked at this with a lot of effort, but I guess [this](https://github.com/rayandrew/with-opencv-action) is the action you can use.

Take this as an hint - maybe it's worth creating another issue for this

**Comment by galeone** [comment]
Great job - with the CI is always better :+1: 

LGTM

**Pull request merged**